src/bokeh/plotting/_figure.py

```diff
@@ -74,7 +74,7 @@
 #-----------------------------------------------------------------------------
 
 #: A default set of tools configured if no configuration is provided
-DEFAULT_TOOLS = "pan,wheel_zoom,box_zoom,save,reset,help"
+DEFAULT_TOOLS = "pan,wheel_zoom,auto_box_zoom,save,reset,help"
```

Comment:
This change needs to be reflected in `_default_tools` in `bokehjs/src/lib/api/figure.ts`.